src/extensions/replay/sessionrecording.ts

```diff
@@ -430,10 +430,8 @@ export class SessionRecording {
     }
 
     /**
-     * Any one trigger can activate the session
-     * So, if they are all the same - return that value
-     * If either is disabled return the other's valu
-     * @private
+     * trigger is active if _either_ URL _or_ event trigger is active
+     * and is pending if _either_ URL _or_ event trigger is pending
```

Comment:
flyby realised this comment was nonsense 🤣

Reply:
Pedantic brain: what if one is active and one is pending?

Reply:
```suggestion
```

also... i'm describing the code... it's a redundant comment